staff/beto-davila/challenges/tic-tac-toe-app/main.js

```diff
@@ -0,0 +1,233 @@
+
+}
+
+cells[6].onclick = function () {
+
+    cells[6].innerText = symbol;
+
+    positionOnTheBoard(symbol, 2, 0);
+
+    handleCheckStatus();
+
+    handleChangeTurn();
+
+}
+
+cells[7].onclick = function () {
+
+    cells[7].innerText = symbol;
+
+    positionOnTheBoard(symbol, 2, 1);
+
+    handleCheckStatus();
+
+    handleChangeTurn();
+
+}
+
+cells[8].onclick = function () {
+
+    cells[8].innerText = symbol;
+
+    positionOnTheBoard(symbol, 2, 2);
+
+    handleCheckStatus();
+
+    handleChangeTurn();
+
+}
+
+
+
+
+// Handle events
+
+
+function positionOnTheBoard(symbol, row, column) {
+    board[row * 3 + column] = symbol;
+
+}
+
+
+
+function handleChangeTurn() {
+
+    if (symbol === "✖") {
+
+        symbol = "◯";
+
+        whoIsPlaying.innerHTML = "Turno de ◯";
+
+    } else {
+
+        symbol = "✖";
+
+        whoIsPlaying.innerHTML = "Turno de ✖";
+    }
+
+}
+
+function handleCheckStatus() {
+
+    if (board[0] === '✖' && board[1] === '✖' && board[2] === '✖' || board[3] === '✖' && board[4] === '✖' && board[5] === '✖' || board[6] === '✖' && board[7] === '✖' && board[8] === '✖') {
+        displayScore.innerHTML = "Gana ✖!";
+
+        return;
+    }
+    else if (board[0] === '◯' && board[1] === '◯' && board[2] === '◯' || board[3] === '◯' && board[4] === '◯' && board[5] === '◯' || board[6] === '◯' && board[7] === '◯' && board[8] === '◯') {
+        displayScore.innerHTML = "Gana ◯!";
+
+        return;
+    }
+    else if (board[0] === '✖' && board[3] === '✖' && board[6] === '✖' || board[1] === '✖' && board[4] === '✖' && board[7] === '✖' || board[2] === '✖' && board[5] === '✖' && board[8] === '✖') {
+        displayScore.innerHTML = "Gana ✖!";
+
+        return;
+    }
+    else if (board[0] === '◯' && board[3] === '◯' && board[6] === '◯' || board[1] === '◯' && board[4] === '◯' && board[7] === '◯' || board[2] === '◯' && board[5] === '◯' && board[8] === '◯') {
+        displayScore.innerHTML = "Gana ◯!";
+
+        return;
+    }
+    else if (board[0] === '✖' && board[4] === '✖' && board[8] === '✖' || board[2] === '✖' && board[4] === '✖' && board[6] === '✖') {
+        displayScore.innerHTML = "Gana ✖!";
+
+        return;
+    }
+    else if (board[0] === '◯' && board[4] === '◯' && board[8] === '◯' || board[2] === '◯' && board[4] === '◯' && board[6] === '◯') {
+        displayScore.innerHTML = "Gana ◯!";
+
+        return;
+    } else {
+
+        handleTieGame ();
+    }
+
+
+
+
+}
+
+
+function handleTieGame() {
+    var count = 0
+
+    for (var i = 0; i < board.length; i++) {
+        if (board[i] !== undefined) {
+            count++
+        }
+    }
+
+    if (count > 7) {
+        displayScore.innerHTML = "EMPATE!";
+    }
+}
+
+resetButton.onclick = function () {
+
+
+    symbol = "✖";
+
+    whoIsPlaying.innerHTML = initialTurn;
+
+    board = [];
+
+    for (let i = 0; i < cells.length; i++) {
+
+        cells[i].innerHTML = "";
+        displayScore.innerHTML = "";
+
+    }
+
+    return;

```

Comment:
empty return is not required if it is at the end. remove it. all functions return by default if you don't explicitly set it.